applications/nrf5340_audio/tools/buildprog/pcft_sign.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
```

Comment:
This file should be run through a Python code formatter like autopep8

Reply:
Will run with autopep8.